systemtests/src/test/java/io/enmasse/systemtest/brokered/jms/TopicTest.java

```diff
@@ -43,26 +47,21 @@ public void setUp() throws Exception {
         addressTopic = Destination.topic(topic);
         setAddresses(addressSpace, addressTopic);
 
-        env = new Hashtable<Object, Object>();
-        env.put(Context.INITIAL_CONTEXT_FACTORY, "org.apache.qpid.jms.jndi.JmsInitialContextFactory");
-        env.put("connectionfactory.qpidConnectionFactory", "amqps://" + getRouteEndpoint(addressSpace).toString() +
-                "?jms.clientID=" + jmsClientID +
-                "&transport.trustAll=true" +
-                "&jms.password=" + jmsUsername +
-                "&jms.username=" + jmsPassword +
-                "&transport.verifyHost=false");
```

Comment:
I think that verifyHost and trustAll should be variable, because there might be in future possibility to send messages on amqp.